public/main.js

```diff
@@ -124,6 +131,7 @@ function reloadTodoList() {
         } else {
             clearCompleteButton.style.visibility = "hidden";
         }
+        todoListRefreshTimerId = setTimeout(reloadTodoList, todoListRefreshRate);
```

Comment:
Your code works, but there is a nicer way of doing this.
You should look into the javascript function setInterval().